server/src/controllers/sessions.controllers.ts

```diff
@@ -0,0 +1,52 @@
+import { AnyAaaaRecord } from "dns";
+import Session from "../models/SessionModel";
+import { Request, Response } from 'express';
+
+const createSession = async (req: Request, res: Response) => {
+    try {
+        const {
+            sessionDate,
+            region,
+            staff,
+            archived,
+            attendance
+        } = req.body;
+
+        const newSession = await Session.create(req.body);
+        await newSession.save();
+
+        return res.status(200).json(newSession);
+    } catch (error: any) {
+        console.error(error.message);
+        return res.status(500).send({ message: error.message });
+    }
+};
+
+const getSessionById = async (req: Request, res: Response) => {
+    const sessionId = req.params.sessionId;
+    try {
+            const session = await Session.findById(sessionId).exec();
```

Comment:
Add an additional if statement to ensure that a 400 error is thrown if sessionId doesn't exist on req.params